packages/core/src/Coordinator.js

```diff
@@ -8,8 +8,7 @@ let _instance;
 
 /**
  * Set or retrieve the coordinator instance.
- *
- * @param {Coordinator} instance the coordinator instance to set
+ * @param {Coordinator} [instance] the coordinator instance to set
```

Comment:
What do the `[]` do here?

Reply:
Oh, it means optional.

Reply:
That's jsdoc syntax to indicate an optional argument, akin to `instance?` in proper TypeScript. Without that, I believe calling the method as a getter will fail type checks.

Reply:
Yeah, I noticed that when I tried `?` for an optional argument. I'm not quite used to jsdocs types yet.